### PR TITLE
Alpaka SiStrip clusterizer

### PR DESCRIPTION
**Pull request by mmasciov**
### PR description:

As per title: draft.

### CODE REVIEW

**Comment by slava77** [comment]
if there's some test and validation material, please attach or add links.
Thanks.

File: CUDADataFormats/SiStripCluster/interface/SiStripUtilities.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+//using clusterADCStartArray = std::array<hindex_type, TrackerTraits::numberOfModules + 1>;
+namespace stripTest {
+struct SiStripSoA {
+
+
+		//using clusterADCsColumn = std::array<uint8_t, 768>;
+
+		GENERATE_SOA_LAYOUT(SiStripClustersSoALayout,
+							SOA_COLUMN(uint32_t, clusterIndex),
```
Comment (slava77) [inline CUDADataFormats/SiStripCluster/interface/SiStripUtilities.h:25]: did this go through the code-checks/code-format?

File: HeterogeneousCore/AlpakaTest/plugins/TestAlpakaAnalyzer.cc [hunk 2]
```diff
@@ -128,7 +128,7 @@ class TestAlpakaAnalyzer : public edm::global::EDAnalyzer<> {
     assert(view.r() == 1.);
     for (int32_t i = 0; i < view.metadata().size(); ++i) {
       auto vi = view[i];
-      assert(vi.x() == 0.);
+      //assert(vi.x() == 0.);
```
Comment (slava77) [inline HeterogeneousCore/AlpakaTest/plugins/TestAlpakaAnalyzer.cc:131]: this and a few more changes in HeterogeneousCore/AlpakaTest don't clearly belong to sistrip developments.
related commits should probably be dropped from history

**Review by slava77 — COMMENTED** [review]
considering some significant changes in functionality (commented out operational code), I guess the degree of "draftness" is quite high, some parts are more like an outline for developments.

**Comment by mmasciov** [comment]
Closed draft PR, will re-open a clean one.

**Pull request closed without merging**